openproblems/tasks/_cell_cell_communication/_common/methods/baseline.py

```diff
@@ -35,7 +35,7 @@ def _random_events(adata, test=False, n_events=1000):
 
 
 @_random_method(
-    method_name="Random Events MAX",
+    method_name="Random Events (max)",
```

Comment:
Now I think about it, we can do away with max/sum aggregation for random events, too -- let's just select them randomly. Since they're already in a random order, we can just take the first.

```
adata.uns["ccc_pred"] = adata.uns["ccc_pred"].loc[~adata.uns["ccc_pred"][adata.uns['merge_keys']].duplicated()]
```